finesse/hardware/dummy_stepper_motor.py

```diff
@@ -1,13 +1,12 @@
 """Code for a fake stepper motor device."""
 import logging
-from typing import Union
 
 from pubsub import pub
 
-from ..config import ANGLE_PRESETS
+from .stepper_motor_base import StepperMotorBase
 
 
-class DummyStepperMotor:
+class DummyStepperMotor(StepperMotorBase):
```

Comment:
The actual controller `ST10Controller` has a bunch of public methods like `wait_until_stopped`, `relative_move`, etc. that are neither present in the dummy nor in the base class `StepperMotorBase`. If they are not to be used externally, then they should have an underscore. If they are meant to be used externally, then the dummy should also have them (or potentially the base class), right?

Reply:
Good point. I'll tweak things